packages/terraform-modules-public/example/variables.tf

```diff
@@ -0,0 +1,278 @@
DA1MTkxNTA2MDFmMTk2MDNmM2QwMTE2ODIwMTYwNDA1MjNkODI1MjNkNjAwMDYwMjA4NDAxM2U2MTAzYzg1NjViNjA2MDkxNTA1YjUwODA5MjUwODE5MzUwNTA1MDgxNjEwNDQ0NTc2MDQwNTE3ZjA4YzM3OWEwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDA4MTUyNjAwNDAxODA4MDYwMjAwMTgyODEwMzgyNTI2MDFlODE1MjYwMjAwMTgwN2Y2OTZlNjk3NDY5NjE2YzY5N2E2MTc0Njk2ZjZlMjA2MzYxNmM2YzYyNjE2MzZiMjA2NjYxNjk2YzY1NjQwMDAwODE1MjUwNjAyMDAxOTE1MDUwNjA0MDUxODA5MTAzOTBmZDViNTA1MDUwNTA1MDU2NWI2MDAwODA2MDQwNTE4MDgwN2Y2ZjcyNjcyZTYzNjU2YzZmMmU2OTZkNzA2YzY1NmQ2NTZlNzQ2MTc0Njk2ZjZlMDAwMDAwMDAwMDAwMDAwMDAwODE1MjUwNjAxNzAxOTA1MDYwNDA1MTgwOTEwMzkwMjA5MDUwODA1NDkxNTA1MDkwNTY1YjYxMDQ5NjYxMDY2ZTU2NWI3M2ZmZmZmZmZmZmZmZmZmZmZmZmZmZmZmZmZmZmZmZmZmZmZmZmZmZmYxNjMzNzNmZmZmZmZmZmZmZmZmZmZmZmZmZmZmZmZmZmZmZmZmZmZmZmZmZmZmMTYxNDYxMDUzNjU3NjA0MDUxN2YwOGMzNzlhMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwODE1MjYwMDQwMTgwODA2MDIwMDE4MjgxMDM4MjUyNjAxNDgxNTI2MDIwMDE4MDdmNzM2NTZlNjQ2NTcyMjA3NzYxNzMyMDZlNmY3NDIwNmY3NzZlNjU3MjAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDgxNTI1MDYwMjAwMTkxNTA1MDYwNDA1MTgwOTEwMzkwZmQ1YjYwMDA2MDQwNTE4MDgwN2Y2ZjcyNjcyZTYzNjU2YzZmMmU2OTZkNzA2YzY1NmQ2NTZlNzQ2MTc0Njk2ZjZlMDAwMDAwMDAwMDAwMDAwMDAwODE1MjUwNjAxNzAxOTA1MDYwNDA1MTgwOTEwMzkwMjA5MDUwODE4MTU1ODE3M2ZmZmZmZmZmZmZmZmZmZmZmZmZmZmZmZmZmZmZmZmZmZmZmZmZmZmYxNjdmYWI2NGY5MmFiNzgwZWNiZjRmMzg2NmY1N2NlZTQ2NWZmMzZjODk0NTBkY2NlMjAyMzdjYTdhOGQ4MWZiN2QxMzYwNDA1MTYwNDA1MTgwOTEwMzkwYTI1MDUwNTY1YjYxMDVjMjYxMDY2ZTU2NWI3M2ZmZmZmZmZmZmZmZmZmZmZmZmZmZmZmZmZmZmZmZmZmZmZmZmZmZmYxNjMzNzNmZmZmZmZmZmZmZmZmZmZmZmZmZmZmZmZmZmZmZmZmZmZmZmZmZmZmMTYxNDYxMDY2MjU3NjA0MDUxN2YwOGMzNzlhMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwODE1MjYwMDQwMTgwODA2MDIwMDE4MjgxMDM4MjUyNjAxNDgxNTI2MDIwMDE4MDdmNzM2NTZlNjQ2NTcyMjA3NzYxNzMyMDZlNmY3NDIwNmY3NzZlNjU3MjAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDgxNTI1MDYwMjAwMTkxNTA1MDYwNDA1MTgwOTEwMzkwZmQ1YjYxMDY2YjgxNjEwNmIxNTY1YjUwNTY1YjYwMDA4MDYwNDA1MTgwODA3ZjZmNzI2NzJlNjM2NTZjNmYyZTZmNzc2ZTY1NzIwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDA4MTUyNTA2MDBlMDE5MDUwNjA0MDUxODA5MTAzOTAyMDkwNTA4MDU0OTE1MDUwOTA1NjViNjAwMDYwNDA1MTgwODA3ZjZmNzI2NzJlNjM2NTZjNmYyZTZmNzc2ZTY1NzIwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDA4MTUyNTA2MDBlMDE5MDUwNjA0MDUxODA5MTAzOTAyMDkwNTA4MTgxNTU4MTczZmZmZmZmZmZmZmZmZmZmZmZmZmZmZmZmZmZmZmZmZmZmZmZmZmZmZjE2N2Y1MDE0NmQwZTNjNjBhYTFkMTdhNzA2MzViMDU0OTRmODY0ZTg2MTQ0YTIyMDEyNzUwMjEwMTRmYmYwOGJhZmUyNjA0MDUxNjA0MDUxODA5MTAzOTBhMjUwNTA1NmZlYTE2NTYyN2E3YTcyMzA1ODIwZDAwMTUxNzVmZTQ3YzM2YWI1ZWRiNTcwMjAyNTg4MTdhZTQwZTlmNWUxOWVkY2YwYWM4NzA2M2ZmOGNmNTlhMzAwMjkiLCJzdG9yYWdlIjp7IjB4MzRkYzVhMjU1NmIyMDMwOTg4NDgxOTY5Njk2ZjI5ZmVkMzhkNDU4MTNkODAwM2Y2YzcwZTVjMTZhYzkyYWUwZiI6IjQ3ZTE3MkY2Q2ZCNmM3RDAxQzE1NzRmYTNFMkJlN0NDNzMyNjlEOTUifSwiYmFsYW5jZSI6IjAifX0sIm51bWJlciI6IjB4MCIsImdhc1VzZWQiOiIweDAiLCJtaXhIYXNoIjoiMHg2Mzc0Njk2MzYxNmMyMDYyNzk3YTYxNmU3NDY5NmU2NTIwNjY2MTc1NmM3NDIwNzQ2ZjZjNjU3MjYxNmU2MzY1IiwicGFyZW50SGFzaCI6IjB4MDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMCJ9"
```

Comment:
I assume we'll plan on fetching this automatically at some point?

Reply:
Yeap, I can add the [http](https://www.terraform.io/docs/providers/http/data_source.html) provider to get it from the bucket. Great idea!